packages/js/src/Modules/Verto/index.ts

```diff
@@ -21,7 +21,68 @@ export default class Verto extends BrowserSession {
     return isValidOptions(this.options);
   }
 
+  /**
+   * Makes a new outbound call.
+   *
+   * This method receives an object `options` with the following properties:
+   *
+   * @param options Options object for a new call.
+   * @param options.destinationNumber Phone number or SIP URI to dial.
+   * @param options.callerNumber Number to use as the caller ID when dialing out to a destination.
+   * @param options.callerName Name to use as the caller ID name when dialing out to a destination.
+   * @param options.id The identifier of the call.
```

Comment:
What does ID do, exactly? Is this a custom UUID you can use to track the call?

Reply:
This is a UUID. 
This becomes a `callID` to track the call

For example:

```
   const newCall = clientRef.current.newCall({
      callerName,
      callerNumber,
      destinationNumber: destination,
      id: 'bb72fb4e-365f-11eb-adc1-0242ac120002',
    });
```

On the backend side you get on was events:

```
{jsonrpc: "2.0", id: 591498, method: "telnyx_rtc.answer",…}
id: 591498
jsonrpc: "2.0"
method: "telnyx_rtc.answer"
params: {callID: "bb72fb4e-365f-11eb-adc1-0242ac120002"}
callID: "bb72fb4e-365f-11eb-adc1-0242ac120002"
```

Reply:
Oh this is great, I didn't even know about this feature.

Reply:
My suggestion to resolve this thread:
```suggestion
   * @param options.id Custom ID to identify the call. This will be used as the `callID` in place of the UUID generated by the client.
```